js/includes/alert.js

```diff
@@ -14,7 +14,19 @@
  */
 function addAlert( message ) {
 	// Add the tribe-marketing-alert class to the body tag so we can style appropriately
-	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-marketing-alert' )
+	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-alert' )
 	// Add the actual message
 	$navbar.after( message )
-}
\ No newline at end of file
+}
+
+/* Remove this when the time comes */
+addAlert( `
+	<div id="tribe-alert">
+		<div class="tribe-alert__content">
```

Comment:
Similarly, I would recommend we use the same class as the example given on line 7:

```suggestion
		<div class="tribe-marketing-alert__content">
```